indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -605,12 +605,14 @@ def get_track_reviewing_state(self, track):
     def get_track_question_scores(self):
         query = (db.session.query(AbstractReview.track_id,
                                   AbstractReviewQuestion,
-                                  db.func.avg(AbstractReviewRating.value))
+                                  db.func.avg(AbstractReviewRating.value.op('#>>')('{}').cast(db.Integer)))
                  .join(AbstractReviewRating.review)
                  .join(AbstractReviewRating.question)
                  .filter(AbstractReview.abstract == self,
+                         AbstractReviewQuestion.field_type == 'rating',
+                         AbstractReviewRating.value.cast(db.String) != db.text("'null'"),
                          ~AbstractReviewQuestion.is_deleted,
-                         ~AbstractReviewQuestion.no_score)
+                         ~AbstractReviewQuestion._no_score)
```

Comment:
if you use the `.expression` property suggested in another comment you can just use `~AbstractReviewQuestion.no_score` here